uktrade/data-hub-frontend
# Referral details page

## Description of change

Trello -https://trello.com/c/owy0yoIf/1113-fe-referral-details-page-6

As part of the referral work this is the referral details page which is currently hidden behind a route that is not exposed anywhere in the app `/referral/{id}`. 

## Test instructions

You will need to post a referral to the back end. Method `POST` `/v4/company-referral/{id}`

For more info see the API docs - https://api.datahub.dev.uktrade.io/docs



Once you have a referral posted just access the referral via `/referral/{id}`

Note: The "Complete referral" button currently does not post anything to the back end, this will be done in another PR.

_What should I see?_

## Screenshots
<img width="1065" alt="Screenshot 2020-02-17 at 19 19 47" src="https://user-images.githubusercontent.com/10154302/74681149-8d9f0f80-51ba-11ea-821f-dd8cf429b545.png">

## Checklist

[//]: # "When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/master/docs/Code%20review%20guidelines.md"
